src/index.html

```diff
@@ -8,6 +8,56 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+
+      <div class="stars__block stars--0">
```

Comment:
```suggestion
      <div class="stars stars--0">
```

`stars__block` is good if you have parent with class `stars`, but you don't 😉